lightning_pose/data/utils.py

```diff
@@ -25,8 +25,28 @@ class BaseLabeledExampleDict(TypedDict):
 
 class HeatmapLabeledExampleDict(BaseLabeledExampleDict):
     """Return type when calling __getitem__() on HeatmapTrackingDataset."""
+    heatmaps: TensorType["num_keypoints", "heatmap_height", "heatmap_width", float]
+
+
+class MultiviewLabeledBatchDict(TypedDict):
+    """Return type when calling __getitem__() on BaseTrackingDataset."""
+    images: Union[
+        TensorType["num_views", "RGB":3, "image_height", "image_width", float],
+        TensorType["num_views", "frames", "RGB":3, "image_height", "image_width", float],
+    ]
+    keypoints: TensorType["num_targets", float]
+    idxs: int
+
+
+class MultiviewHeatmapLabeledBatchDict(MultiviewLabeledBatchDict):
```

Comment:
change *Batch* to *Example* in both class name and inheritance

Reply:
will then need to add, below `HeatmapLabeledBatchDict`,
```
MultiviewLabeledBatchDict
```
and
```
MultiviewHeatmapLabeledBatchDict
```
which will mostly look like the above, but with additional "batch" dimensions on the tensors